Web_Development/FrontEnd/HTML_Difference_between_HTML_&_HTML5/README.md

```diff
@@ -0,0 +1,40 @@
+# Difference between HTML and HTML5
+
+
+### What is HTML AND HTML5
+
+- HTML:- 
+Also known as the Hyper Text Markup Language. It is the skeleton like a human body on which different elements can fitted to make a webpage. Mainly used to design web pages. Hypertext defines the link between the web pages .The language uses tags to define what changes need to be done on the text within the tags eg:-  the p tag for displaying the content in a paragraph format, the strong tag for displaying it in a bold format. It is used for structuring and presenting the content on the web pages.
+ 
+ - HTML5 :- It is nothing but the latest or the fifth version of HTML. It has many added features with old features either removed or modified.
+ 
+### Difference
+Lets compare them on the basis of some points
+##### Browser Compatibility-
+A great way of explaining it would be, If it works in my browser then it is html5 and if it does not then it is not html5 because html5 is supported by all newbrowsers like      Safari,Chrome,Mozilla etc. On the other hand html works in old browsers.
+##### Storage- 
+Cookies were used to store temporary data earlier but now sql databases,application cache are now used to store data. 
+##### Multimedia Support- 
+In Older versions of html, audio and video were supported only with the use of adobe flash player support while Html5 now supports the use of audio and video tags 
+##### Vector Graphics- 
+It was possible with the help of technologies like VML,SilverLight,Flash etc in old versions of HTML but now in Html5 it is an integral part of Html5,svg and canvas.
+##### Running JavaScript- 
+JavaScript can be now run in the background due to JS Web worker API in HTML5 earlier it is cannot be executed in the browser. 
+##### Drawing Shapes- 
+Html5 now allows us to draw shapes like rectangle,circle,triangle etc.
+##### Error handling-
+Html couldn't handle inaccurate syntax but html5 can!
+##### Doctype Declaration- 
```

Comment:
Don't add ##### in each and every point. Write it normally please it's looking clustered.